### ISSUE TITLE
Backend service fails to start v1.7.1

### ISSUE THREAD

**Issue by ruskid** [madewithkoji/koji-core#44]
I am getting this error when trying to start the backend service. It was working fine with the 1.4.0 version but now fails using the latest: ^1.7.1

```
➜  backend git:(master) ✗ npm run start-dev

> backend@1.0.0 start-dev /Users/demin/Projects/Koji/ExpensiveChat/backend
> nodemon --exec babel-node ./src/server.js

[nodemon] 2.0.7
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): *.*
[nodemon] watching extensions: js,mjs,json
[nodemon] starting `babel-node ./src/server.js`
/Users/demin/Projects/Koji/ExpensiveChat/backend/node_modules/@withkoji/core/dist/frontend/index.js:75
    if (!window.KOJI_FEED_KEY) {
    ^

ReferenceError: window is not defined
    at new Koji (/Users/demin/Projects/Koji/ExpensiveChat/backend/node_modules/@withkoji/core/src/frontend/index.ts:98:5)
    at Object.<anonymous> (/Users/demin/Projects/Koji/ExpensiveChat/backend/node_modules/@withkoji/core/src/frontend/index.ts:364:16)
    at Module._compile (internal/modules/cjs/loader.js:1063:30)
    at Module._compile (/Users/demin/Projects/Koji/ExpensiveChat/backend/node_modules/pirates/lib/index.js:99:24)
    at Module._extensions..js (internal/modules/cjs/loader.js:1092:10)
    at Object.newLoader [as .js] (/Users/demin/Projects/Koji/ExpensiveChat/backend/node_modules/pirates/lib/index.js:104:7)
    at Module.load (internal/modules/cjs/loader.js:928:32)
    at Function.Module._load (internal/modules/cjs/loader.js:769:14)
    at Module.require (internal/modules/cjs/loader.js:952:19)
    at require (internal/modules/cjs/helpers.js:88:18)
[nodemon] app crashed - waiting for file changes before starting...

```

**Comment by sthielen** [comment]
hey - sorry about that! fix should be out in a few min

**Comment by sthielen** [comment]
this should be fixed in v1.7.2!
